soundbay/inference.py

```diff
@@ -134,8 +134,45 @@ def inference_to_file(
     output_file = output_path / filename
     concat_dataset.to_csv(index=False, path_or_buf=output_file)
 
+    # save raven file
+    threshold = 0.5
```

Comment:
There is no use in this one since You define a default value in `inference_csv_to_raven` function.

Reply:
that's correct. Perhaps we should have inference args conf?

Reply:
I added it to the conf file